modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -6,64 +6,140 @@
-<1> Add NMState YAML syntax to configure host interfaces.
++
+Replace `<NIC1_name>`, `<IP_address>`, `<DNS_IP_address>`, `<next_hop_IP_address>` and `<next_hop_NIC1_name>` with appropriate values.
 
-[TIP]
-====
-Consider saving the `networkConfig` YAML syntax to a file and testing it using the NMState command line interface before including it in the `install-config.yaml` file, because the installer will not check the NMState YAML syntax. Execute `nmstatectl gc <yaml-config>` to test the syntax. Errors in the YAML syntax might result in a failure to apply the network configuration. Additionally, maintaining the validated YAML syntax is useful when applying changes using Kubernetes NMState after deployment or when expanding the cluster.
-====
-
-The most common use case for this functionality is to specify a static IP address on the `baremetal` network, but you can also configure other networks such as a storage network. This functionality will also support other NMState features such as VLAN, VXLAN, bridges, bonds, routes, MTU, and DNS resolver settings.
+.. Test the configuration file by running the following command:
++
+[source,terminal]
+----
+$ nmstatectl gc <networkConfig-configuration-file>
+----
 
+. To use the `networkConfig` configuration setting, add the `NMState` configuration within `install-config.yaml`, as shown in the following example:
++
+[source,terminal]
+----
+    hosts:
+      - name: openshift-master-0
+        role: master
+        bmc:
+          address: redfish+http://<out_of_band_ip>/redfish/v1/Systems/
+          username: <user>
+          password: <password>
+          disableCertificateVerification: null
+        bootMACAddress: <NIC1_mac_address>
+        bootMode: UEFI
+        rootDeviceHints:
+          deviceName: "/dev/sda"
+        networkConfig: <1>
+          interfaces:
+          - name: <NIC1_name>
+            type: ethernet
+            state: up
+            ipv4:
+              address:
+              - ip: <IP_address>
+                prefix-length: 24
+              enabled: true
+          dns-resolver:
+            config:
+              server:
+              - <DNS_IP_address>
+          routes:
+            config:
+            - destination: 0.0.0.0/0
+              next-hop-address: <next_hop_IP_address>
+              next-hop-interface: <next_hop_NIC1_name>
+----
+<1> Add NMState YAML syntax to configure host interfaces.
++
 [IMPORTANT]
 ====
 Once deployed, you cannot modify the `networkConfig` configuration setting of `install-config.yaml` file to make changes to the host network interface. Use the Kubernetes NMState Operator to make changes to the host network interface after deployment.
 ====
+
+. Optional: In the case of deploying {product-title} without a DHCP server on the `baremetal` network, use the following procedure to configure a static IP address for the bootstrap VM using Ignition:
+
+.. Create the `bootstrap_config.sh` file:
++
+[source,terminal]
+----
+#!/bin/bash
```

Comment:
There is a lot of detail in this Bash script. It would be useful for the reader to add call outs explaining each element. You could alternatively consider splitting the steps in the Bash script out into individual commands with descriptions about what each step does.

Reply:
Actually, that's how it was. Engineering wanted it consolidated to reduce the number of steps.